.rubocop.yml

```diff
@@ -13,7 +13,25 @@ Metrics/BlockLength:
 ModuleLength:
   Enabled: false
 
-Style/EmptyMethod:
+Style/GlobalVars:
```

Comment:
Can you please create a separate .rubocop.yml in the spec/ folder for disabling these rules? That way the rules will be disabled only for specs. Use `inherit_from` for inheritance. For more info, see this: http://rubocop.readthedocs.io/en/latest/configuration/